package.json

```diff
@@ -15,9 +15,21 @@
     "updated": "lerna updated",
     "watch": "tsc --build --watch"
   },
+  "dependencies": {
+    "npm-which": "^3.0.1"
```

Comment:
Please put development dependencies as "devDependencies". This helps tools like dependabot determine if the runtime is affected by vulnerabilities or not, or only development tools.

Reply:
Dependencies should have been restored as they were.